.github/workflows/release-branch-tagger.yml

```diff
@@ -0,0 +1,27 @@
+name: Release Branch Tagger
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - 'release/**'
+
+jobs:
+  release-branch-tagger:
+    name: Tag & Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          # Action needs all info about tags so pass fetch-depth: 0
+          fetch-depth: 0
+
+      - name: Tag & Release
+        uses: ./release-branch-tagger # Uses an action in the root directory
```

Comment:
Sample usage of workflow file that will need to be included in the repos.

Note: uses will need to be adjusted to: `skedify/github-workflows/release-branch-trigger@develop`